data/languages.json

```diff
@@ -410,8 +413,6 @@
   "osa": {},
   "ota": {},
   "pa": {"nativeName": "ਪੰਜਾਬੀ"},
-  "pa-Arab": {"base": "pa", "script": "Arab"},
-  "pa-Guru": {"base": "pa", "script": "Guru"},
```

Comment:
Should `name:pa-Arab` and `name:pa-Guru` be marked as deprecated in [deprecated.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/deprecated.json) in the id-tagging-schema repository? That will make it easier for mappers to migrate these keys over to the ones you’re recommending without having to hunt for them in the raw tag editor.